src/chainstate/stacks/db/mod.rs

```diff
@@ -1205,6 +1208,27 @@ impl StacksChainState {
                 callback(&mut clarity_tx);
             }
 
+            // Setup burnchain parameters for pox contract
```

Comment:
Minor nit -- can you put this *before* the call to `post_flight_callback`?   This code used to be the post-flight callback; we might even consider removing the `post_flight_callback` now that the code lives here.